### PR TITLE
Add interactive notebook and requirements

### PR DESCRIPTION
**Pull request by jungaretti**
Adds a new notebook that reads data from a CSV file.

Screen Shot 2022-10-26 at 5 25 51 PM

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
 	},
 	"customizations": {
 		"codespaces": {
-			"openFiles": ["notebooks/Linear.ipynb"]
+			"openFiles": [
+				"notebooks/population.ipynb"
```
Comment (tanmayeekamath) [inline .devcontainer/devcontainer.json:8]: Not sure why, but this command loads a slightly different format notebook on creation
Screenshot: 
image

Is that expected?

**Review by tanmayeekamath — CHANGES_REQUESTED** [review]
Added one comment around the notebook not loading correctly on creation.
Everything else looks great! Thanks for adding this in :)

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (jungaretti) [reply .devcontainer/devcontainer.json:8]: I'm able to repro by creating a new codespace for this branch and opening it in the web.

http://codespaces.new/?repo=github/codespaces-jupyter&ref=jungaretti/add-requirements

Comment (jungaretti) [reply .devcontainer/devcontainer.json]: This looks like a bug with `openFiles`. I am going to remove it from this PR and open an issue

Comment (jungaretti) [reply .devcontainer/devcontainer.json]: https://github.com/github/codespaces/issues/10589

**Pull request merged**